### PR TITLE
test collection with fields

### PR DESCRIPTION
**Pull request by ThreadDao**
-  test collection with fields
- test collection dup drop

Covers: #5312 #5317 #5331

Signed-off-by: ThreadDao yufen.zong@zilliz.com

### CODE REVIEW

File: tests20/python_client/base/client_request.py [hunk 1]
```diff
@@ -108,5 +108,5 @@ def _collection(self, name=None, data=None, schema=None, check_res=None, **kwarg
         self._connect()
         name = cf.gen_unique_str("ApiReq") if name is None else name
         schema = cf.gen_default_collection_schema() if schema is None else schema
-        collection = self.collection.collection_init(name=name, data=data, schema=schema, check_res=check_res, **kwargs)
+        collection, _ = self.collection.collection_init(name=name, data=data, schema=schema, check_res=check_res, **kwargs)
         return name, collection
```
Comment (yanliang567) [inline tests20/python_client/base/client_request.py:112]: "return collection " is enough i think.

File: tests20/python_client/common/common_func.py [hunk 2]
```diff
@@ -103,6 +103,19 @@ def get_vectors(num, dim, is_normal=True):
     return vectors.tolist()
 
 
+def gen_invalid_field_types():
+    field_types = [
+        6,
```
Comment (yanliang567) [inline tests20/python_client/common/common_func.py:108]: 6 and 1.0 are invalid?

File: tests20/python_client/base/client_request.py (hunk 1 above)
Comment (ThreadDao) [reply tests20/python_client/base/client_request.py]: ok. You decide.

File: tests20/python_client/common/common_func.py (hunk 2 above)
Comment (ThreadDao) [reply tests20/python_client/common/common_func.py:108]:
```
class DataType(IntEnum):
    NONE = 0
    BOOL = 1
    INT8 = 2
    INT16 = 3
    INT32 = 4
    INT64 = 5

    FLOAT = 10
    DOUBLE = 11

    STRING = 20

    BINARY_VECTOR = 100
    FLOAT_VECTOR = 101

    UNKNOWN = 999
```

**Review by yanliang567 — APPROVED** [review]

**Pull request merged**